src/index.js

```diff
@@ -107,90 +137,23 @@ async function authenticate(username, password) {
   }
 }
 
-function fetchFiles(year) {
-  log('info', `Fetching files paie for year ${year}`)
-  return request.get(`${baseUrl}/prive/remunerationpaie/v1?annee=${year}`, {
-    json: true
+async function getDocs() {
+  await request({
+    uri: `${baseUrl}/prive/initialiserhabilitation/v1`,
+    method: 'POST',
+    body: {},
+    resolveWithFullResponse: true
   })
-}
-
-function fetchFilesPension(year) {
-  log('info', `Fetching files pension for year ${year}`)
-  return request.get(`${baseUrl}/prive/remunerationpension/v1?annee=${year}`, {
-    json: true
+  let resp = await request({
+    uri: `${baseUrl}/prive/accueilconnecte/v1`,
+    resolveWithFullResponse: true
```

Comment:
I don't think you need resolveWithFullResponse here since you only use the body

Reply:
True, it will be removed